src/auslib/web/admin/swagger/api_v2.yml

```diff
@@ -139,40 +176,158 @@ paths:
     put:
       summary: Overwrites an entire release
       operationId: auslib.web.admin.views.releases_v2.set_release
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/ReleaseModifyBody'
+                - type: object
+                  properties:
+                    # Required if the Release doesn't yet exist, but it's not possible to encode this into the schema
+                    product:
+                      type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
I can certainly include something here. The main reason I avoided this was because the full structure of the return value varies based on the Blob schema. You're probably right that it's still worthwhile to do though.